lib/asciidoctor/opal_ext.rb

```diff
@@ -7,6 +7,10 @@
   // or we can check for document
   //else if (typeof document !== 'undefined' && document.nodeType) {
     value = 'browser';
+
+    if(typeof navigator != 'undefined' && navigator.appVersion)
+       if(navigator.appVersion.indexOf('JavaFX') != -1)
+          value = 'javafx'
```

Comment:
As you can see, other code in this file has one space after `if` and block is always wrapped in braces. Please reformat it to:

```javascript
if (typeof navigator != 'undefined' && navigator.appVersion) {
  if (navigator.appVersion.indexOf('JavaFX') != -1) {
    value = 'javafx'
  }
}
```